scripts/utils.sh

```diff
@@ -34,17 +34,17 @@ service ${service_name}
   per_source	= UNLIMITED
 }
 EOF
-
-  systemctl start xinetd
-  systemctl reload xinetd
+  sudo mv build/xinetd-${service_name} /etc/xinetd.d/${service_name} --force
+  sudo systemctl start xinetd
+  sudo systemctl reload xinetd
 }
 
 function run_in_background() {
   bash -c "nohup $1  >/dev/null 2>&1 &"
 }
 
 function kill_all_port_forwardings() {
```

Comment:
Unused. Can be deleted. (Update Makefile rule and README accordingly)

Reply:
i think will use it later, didn't wanted to delete